base/src/main/java/uk/ac/ebi/atlas/model/experiment/baseline/Cell.java

```diff
@@ -0,0 +1,24 @@
+package uk.ac.ebi.atlas.model.experiment.baseline;
+
+import uk.ac.ebi.atlas.model.BiologicalReplicate;
+import uk.ac.ebi.atlas.model.DescribesDataColumns;
+
+import java.util.Set;
+
+public class Cell extends DescribesDataColumns {
+
+    public Cell(String id) {
+        super(id);
+    }
+
+
+    @Override
+    public Set<String> assaysAnalyzedForThisDataColumn() {
+        return null;
```

Comment:
This, as well as the method below, seems quite dangerous. I strongly suggest returning an immutable empty set. If you want to flag this method as something that shouldn’t be called, throw a non-checked (i.e. wrapped) `OperationNotSupportedException`.